api.lua

```diff
@@ -95,36 +95,55 @@ function areas:getAreasIntersectingArea(pos1, pos2)
 	return res
 end
 
--- Checks if the area is unprotected or owned by you
+-- Returns smallest area at position and its id or nil.
+-- If multiple areas have the same volume, larger id takes precedence.
+function areas:getSmallestAreaAtPos(pos)
+	local smallest_area, smallest_id, smallest_volume, volume = nil
+	for id, area in pairs(self:getAreasAtPos(pos)) do
+		volume =	(area.pos2.x - area.pos1.x + 1)
+					* (area.pos2.y - area.pos1.y + 1)
+					* (area.pos2.z - area.pos1.z + 1)
+		if not smallest_volume or smallest_volume >= volume then
```

Comment:
```suggestion
		if smallest_volume >= volume then
```

Reply:
Done in https://github.com/minetest-mods/areas/pull/79/commits/56763e16f9b262059782935c27adb91bd84a3c99